### ISSUE TITLE
Support all API parameters?

### ISSUE THREAD

**Issue by theDanielJLewis** [DualDev/node-vultr#8]
Am I correct in understanding this doesn't support the full range of Vultr's API parameters?

For `.server.create()`, for example, the documentation shows only support for OS, region, and plan ID, but it doesn't show support for [any of the other parameters the API offers](https://www.vultr.com/api/#server_create), like SSH keys, startup scripts, and other server options.

**Comment by DeviaVir** [comment]
Yes, correct. I've added what I needed and opensourced it so others could extend it with whatever they used.
